### ISSUE TITLE
[BUG] - cannot use `transaction build` command to build Mary era Tx in Babbage

### ISSUE THREAD

**Issue by mkoura** [IntersectMBO/cardano-node#4287]
*Internal*

**Area**
*Other*   Any other topic (Delegation, Ranking, ...).

**Summary**
Building a Tx using `transaction build --mary-era` fails with
```
Command failed: transaction build  Error: "minUtxoValue" field not present in protocol parameters when trying to calculate minimum UTxO value.
```

**Expected behavior**
Since https://github.com/input-output-hk/cardano-node/pull/4135 is merged, it should be possible to use `transaction build` for building Tx from any past era.
Probably low prio, it is possible to use `transaction build-raw`.

**System info (please complete the following information):**
- OS Name: Fedora 35
- Node version: 1.35.3-rc1
- CLI version: 1.35.3-rc1

**Comment by Jimbo4350** [comment]
Is this still an issue? @mkoura

**Comment by mkoura** [comment]
Yes, still an issue with latest master (d15ff2b736452857612dd533c1ddeea2405a2630).
